spec/models/widget/card_spec.rb

```diff
@@ -45,4 +45,36 @@
       expect(Widget::Card.body).not_to include(page_card)
     end
   end
+
+  context "when is not header card" do
```

Comment:
How about moving this whole block to the "validations" section of this file?

By the way:

```suggestion
  context "regular cards" do
```